src/server/controllers/managers.js

```diff
@@ -48,13 +48,16 @@ exports.getDetails= async (req, res) => {
         id: managerId 
       }
     });
-    const details= (result[0].dataValues)
     if (result[0]) {
-      res.send({ details });
+      const details= (result[0].dataValues);
+      console.log(details);
+      
+      res.status(200).send(details);
     } else {
-      res.send({ message: "Error in finding result" });
+      res.status(404).send("Error in finding result");
     }
   } catch (error) {
-    res.send({ error });
+    console.log(error)
+    res.status(500).send('Server Error');
   }
```

Comment:
Nice job with the error codes here, the 404 and 500 make sense for those two error cases.